container-disc/src/main/java/com/yahoo/container/usability/BindingsOverviewHandler.java

```diff
@@ -99,6 +104,76 @@ static JSONArray renderRequestHandlers(JdiscBindingsConfig bindingsConfig,
         return ret;
     }
 
+    static JSONArray renderLinks(URI requestUri,
+                                 JdiscBindingsConfig bindingsConfig,
+                                 Map<ComponentId, ? extends RequestHandler> handlersById)
+    {
+        var entryPointSet = new LinkedHashSet<URI>();
```

Comment:
What do you think about having these URIs sorted lexically?

Reply:
I don't know! :)

Reply:
Ok, let's to the current ordering for the initial approach.